packages/main/src/Menu.ts

```diff
@@ -263,22 +266,68 @@ class Menu extends UI5Element {
 		return this.shadowRoot!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
 	}
 
+	get _list() {
+		return this.shadowRoot!.querySelector<List>("[ui5-list]")!;
```

Comment:
List is conditionally rendered so we can't use ! here

Reply:
Done